github-sync/github-data/repositories.yaml

```diff
@@ -260,6 +260,38 @@ repositories:
         requiredApprovingReviewCount: 1
         statusChecks:
           - DCO
+  - name: docs
+    owner: sigstore
+    description: Sigstore documentation
+    homepageUrl: https://sigstore.dev
```

Comment:
Thanks, Carlos! This will eventually point to the docs.sigstore.dev subdomain. I manually added branch protection, do you want me to change anything there?

Reply:
@ltagliaferri thanks Lisa, this automation should add the branch protection then I don't need to import and manually track it, I will remove the branch protection in favor of this PR.

when we have the subdomain we update it here :)

Reply:
I removed the manual protection, thanks!